### PR TITLE
docs: #38 address issue missing token info

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

**Fixes:** #38 

## Summary:

This PR addresses missing information for setting up WCSS and its dependency on Design Tokens. 

## Type of change:

Please delete options that are not relevant.

- [ ] New capability 
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Other (please elaborate) 

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [ ] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._ 

**Thank you for your submission!**<br>
-- Orion Design System Team

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +148,50 @@ none, xxxs, xxs, xs, sm, md, lg, xl, xxl, xxxl
...
+#### Deprecated files
+
+| deprecated file | new file | description |
+|---|---|---|
+| baselineLTE.scss | core.scss | File deprecation to support new features |
+| baseline.scss | essentials.scss | `essentials.scss` does not import headings by default. For new Auro heading sselectors import `headings.scss`. These new selectors support scoping and prefixing.
+| inputTypeText.scss | n/a | n/a |
+| _layoutProperties.scss | see `_insetUtility.scss` `_layoutPropertiesGenerator.scss` and `_spacingUtility.scss` | The legacy layoutProperties file was broken up into separate concerns |
+
+#### Missing tokens?
```
Comment (dlesaca) [inline README.md]: Should we make a section called "Troubleshooting" or something to that effect and make the "Missing ___" sub-sections of that? I think most docs would benefit from better structuring/organization.

File: README.md [hunk 2]
```diff
@@ -16,54 +16,20 @@ This repository is a library of core level styles, functions, and mixins that ca
...
-```
-
-#### Missing layout selectors?
-
-If you have been using selectors from the now deprecated `_layoutProperties.scss` file, you may be missing the following selectors from your CSS output
-
-```css
-.util_[margin/padding][Top/Right/Bottom/Left]--[none/xs/md/lg/xl]
-```
+## Install
```
Comment (dlesaca) [inline README.md]: Is this as comprehensive as it needs to be? Are there any missing steps?

Comment (blackfalcon) [reply README.md]: Yes. The scope of the install is covered and all use-cases where additional instruction is required.

File: README.md (hunk 1 above)
Comment (blackfalcon) [reply README.md]: Moved content from under DEPRECATED to TROUBLESHOOTING and listed items as FAQs.

**Comment by blackfalcon** [comment]
@dlesaca the issues you pointed out have been addressed. Overall I agree with the comments about better content structure, but that work is outside the scope of this issue. 

Please create a new issue with your recommendations that can be prioritized at a later date.

**Pull request merged**